scss/bootstrap/vendor/select2/select2-bootstrap.scss

```diff
@@ -1,94 +1,250 @@
-.form-control .select2-choice {
+.form-control.select2-container,
+.form-group .select2-container {
+  background: $crm-white;
+  border: 1px solid $crm-darker-grayblue;
+  box-shadow: $crm-form-control-shadow;
+  height: 30px;
+  padding: 0;
+
+  .select2-choice {
     border: 0;
     border-radius: 2px;
-    text-decoration: none !important;
-}
+    text-decoration: none!important;
```

Comment:
@boriszaydlin 
Not sure why the beautifier removed the space between !important and none. what is the correct format?